contracts/AnchoredPriceView.sol

```diff
@@ -185,17 +187,27 @@ contract AnchoredPriceView {
         }
     }
 
+    /**
+     * @notice Returns price denominated in USD, with 6 decimals
+     * @dev If price was posted by source, return it. Otherwise, return anchor price converted through source ETH price.
+     */
     function prices(string calldata symbol) external view returns (uint64) {
         uint64 price = _prices[symbol];
-        // If price was posted by source, eturn it. Othrwise, return anchor price
+
         if (price != 0) {
             return price;
         } else {
-            uint256 usdcPrice = anchor.getUnderlyingPrice(cUsdcAddress);
-            return getAnchorPrice(getCTokenAddress(symbol), usdcPrice);
+            uint256 usdPerEth = _prices["ETH"];
+            uint256 ethPerToken = anchor.getUnderlyingPrice(getCTokenAddress(symbol));
+
+            // ethPerToken has 18 decimals since the usdt, usdc, wbtc tokens hit
+            // usdPerEth has 6 decimals
+            // scaling by 1e18 and dividing leaves 1e6, as desired
+            return uint64(mul(usdPerEth, 1e18) / ethPerToken);
```

Comment:
probably won't matter in the real world, but I thought it was tighter for the "source" eth price to control the values here rather than anchor "usdc" price